buildSrc/src/main/groovy/com/uber/okbuck/OkBuckGradlePlugin.groovy

```diff
@@ -99,11 +93,11 @@ class OkBuckGradlePlugin implements Plugin<Project> {
         // Create tasks
         Task setupOkbuck = project.task('setupOkbuck')
         setupOkbuck.setGroup(GROUP)
-        setupOkbuck.setDescription("Setup okbuck cache and dependencies")
+        setupOkbuck.setDescription()
 
-        Task okBuck = project.task(OKBUCK)
-        okBuck.setGroup(GROUP)
-        okBuck.setDescription("Generate BUCK files")
+        Task okBuck = project.tasks.create(OKBUCK, OkBuckTask, {
```

Comment:
okbuck task is not safe to be an incremental task. You should set it as never up to date (which `doLast` does as a side effect). When it is truly incremental, we can lift that restriction